Source/WebCore/css/makeprop.pl

```diff
@@ -689,7 +689,8 @@ sub sortByDescendingPriorityAndName
 {
     switch (id) {
 EOF
-for my $logicalPropertyGroup (sort values %logicalPropertyGroups) {
```

Comment:
Why doesn't `sort values` make it deterministic?

Reply:
I'm no `perl` expert 😅 but AFAICT the values themselves are some kind of objects (dictionaries?) and `sort` is not doing something deterministic with them.

Reply:
Makes sense.